BibBuddy/app/src/main/java/de/bibbuddy/BookFragment.java

```diff
@@ -128,6 +130,7 @@ public void handleOnBackPressed() {
     shelfId = bundle.getLong(LibraryKeys.SHELF_ID);
 
     bookModel = new BookModel(getContext(), shelfId);
+    noteModel = new NoteModel(getContext());
```

Comment:
```suggestion
    noteModel = new NoteModel(requireContext());
```
Replace all getContext() -usages with requireContext() to avoid NullPointerExceptions.